src/lib/bootloader/autoinst_profile/bootloader_section.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "installation/autoinst_profile/section_with_attributes"
+
+module Bootloader
+  module AutoinstProfile
+    # This class represents an AutoYaST <bootloader> section
+    #
+    class BootloaderSection < ::Installation::AutoinstProfile::SectionWithAttributes
```

Comment:
this looks a bit strange for me as it is not used for anything except issues. What about allowing to pass hash to issues and create instance of SectionWithAttributes there for Hash? Or adapt code to use it at more places instead of plain hash.

Reply:
On of the objectives of the AutoYaST refactor is to improve how the profile is represented (using hashes everywhere is not that nice). So I will take the opportunity to extend this `BootloaderSection` so it can be used when importing a profile instead of a hash.

As far as I can see, the hash is used only in [::Bootloader::AutoyastConverter](https://github.com/yast/yast-bootloader/blob/038d3eecd16a6d24836b28a667723e2287eff5b8/src/lib/bootloader/autoyast_converter.rb), so perhaps we can  adapt that module